test/ExecutionFramework.cpp

```diff
@@ -157,17 +160,22 @@ void ExecutionFramework::sendMessage(bytes const& _data, bool _isCreation, u256
 			std::cout << "CALL   " << m_sender.hex() << " -> " << m_contractAddress.hex() << ":" << std::endl;
 		if (_value > 0)
 			std::cout << " value: " << _value << std::endl;
-		std::cout << " in:      " << util::toHex(_data) << std::endl;
+		std::cout << " in:      " << util::toHex(_bytecode + _arguments) << std::endl;
```

Comment:
You have `_data` defined above, why not use it?
```suggestion
		std::cout << " in:      " << util::toHex(_data) << std::endl;
```

Otherwise I'd move it to the `if` below - no need to calculate it if you won't use it.

Reply:
Also, `_data` -> `data`. This is no longer a parameter.